README.md

```diff
@@ -238,7 +236,7 @@ source ~/ros2_mara_ws/install/setup.bash
 rviz2 -d `ros2 pkg prefix mara_description`/share/mara_description/rviz/visualization.rviz
 ```
 
-Alternatively, instead of using the `robot_description` topic, you can load the 3D model manually selecting the URDF file in RViz.
+Alternatively, instead of using the `robot_description` topic, you can load the 3D model manually selecting the URDF file in the RobotModel.
```

Comment:
Can you change only `RViz` for `the RobotModel section in RViz` ? I would discard all the other changes.

Reply:
Done